nuxt.config.js

```diff
@@ -9,18 +9,23 @@ export default {
       { name: 'viewport', content: 'width=device-width, initial-scale=1' },
       { hid: 'description', name: 'description', content: '' },
     ],
-    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.png' }],
+    link: [
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.png' }
+    ],
   },
 
   // Parts of our css that are globalF
   css: [
     '@/assets/_typography',
     '@/assets/_shadows',
     '@/assets/_vuetifycomponents',
+    '@mdi/font/css/materialdesignicons.min.css',
   ],
 
   // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
-  plugins: [],
+  plugins: [
+    { src: '@/plugins/vue-awesome-swiper', mode: 'client' },
```

Comment:
in general this nuxt swiper is a very nice solution for this mini project, on bigger scale you should avoid adding global plugins at all cost and try to load it only in the components, which need it